### ISSUE TITLE
About using pretrained MedCLIP for fine-tuning on downstream tasks

### ISSUE THREAD

**Issue by foxofag001** [RyanWangZf/MedCLIP#36]
hello,
First, thanks for creating MedCLIP. It's a great job.
In your published paper on this work, experiments were done and the answer to Q3: Does MedCLIP lead to better performance and label efficiency for downstream classification tasks through fine-tuning? I want to use the MedCLIP model to fine-tune my dataset downstream. Can you share a copy of the code for fine-tuning on a downstream task? Thank you very much!

best wishes，
Hu Tian
